src/mono/mono/metadata/native-library.c

```diff
@@ -1272,7 +1272,7 @@ lookup_pinvoke_call_impl (MonoMethod *method, MonoLookupPInvokeStatus *status_ou
 			new_scope = g_strdup ("libcoreclr.dylib");
 #else			
 #if defined(TARGET_ANDROID)
-			new_scope = g_strdup ("libmonosgen-2.0.so");
+			new_scope = g_strdup ("libruntime-android.so");
```

Comment:
NOTE: I'll rename it later to the one XA uses (and I'll rename our runtime for tests) - `libmonodroid.so` as far as I remember